packages/studio-ui/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@yext/studio-ui",
+  "version": "0.22.0",
+  "types": "./lib/src/index.d.ts",
+  "main": "./lib/src/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./lib/src/index.d.ts",
+      "import": "./lib/src/index.js",
+      "require": "./lib/src/index.js"
+    }
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "dev": "npm run copy-svg && tsc --watch --preserveWatchOutput -p tsconfig.json",
+    "copy-svg": "npx copyfiles \"src/**/*.svg\" lib",
+    "build": "rimraf lib && npm run copy-svg && tsc -p tsconfig.json",
+    "test": "jest",
+    "typecheck-jest": "npx tsc -p tests/tsconfig.json"
+  },
+  "dependencies": {
+    "@dhmk/zustand-lens": "^2.0.5",
+    "@minoru/react-dnd-treeview": "^3.4.1",
+    "@restart/ui": "^1.5.2",
+    "@vitejs/plugin-react": "^4.0.4",
+    "autoprefixer": "^10.4.14",
+    "cac": "^6.7.14",
+    "classnames": "^2.3.2",
+    "cross-env": "^7.0.3",
+    "immer": "^9.0.21",
+    "lodash": "^4.17.21",
+    "path-browserify": "^1.0.1",
+    "postcss": "^8.4.27",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-modal": "3.16.1",
+    "react-toastify": "^9.1.1",
+    "react-tooltip": "^5.18.0",
+    "tailwind-merge": "^1.8.1",
+    "tailwindcss": "^3.3.3",
+    "vite": "^4.4.7",
+    "vite-plugin-svgr": "^2.4.0",
+    "zundo": "2.0.0-beta.12",
+    "zustand": "^4.3.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.5",
+    "@babel/plugin-syntax-flow": "^7.18.6",
+    "@babel/plugin-transform-react-jsx": "^7.19.0",
+    "@rollup/plugin-typescript": "^10.0.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/jest": "^29.2.4",
+    "@types/lodash": "^4.14.191",
+    "@types/node": "^18.11.15",
+    "@types/path-browserify": "^1.0.0",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.10",
+    "@types/react-modal": "3.13.1",
+    "@yext/studio-plugin": "*",
+    "copyfiles": "^2.4.1",
```

Comment:
I see that we might have some new dev dependencies-- should we regenerate the third party notices to include those, and for studio-ui?